projects/example-app/src/app/books/containers/selected-book-page.component.ts

```diff
@@ -25,11 +25,9 @@ export class SelectedBookPageComponent {
   isSelectedBookInCollection$: Observable<boolean>;
 
   constructor(private store: Store<fromBooks.State>) {
-    this.book$ = store.pipe(select(fromBooks.selectSelectedBook)) as Observable<
-      Book
-    >;
-    this.isSelectedBookInCollection$ = store.pipe(
-      select(fromBooks.isSelectedBookInCollection)
+    this.book$ = store.select(fromBooks.selectSelectedBook) as Observable<Book>;
```

Comment:
I wonder why the cast is needed here 🤔

Reply:
You are thinking what I was thinking when I saw it. Since this was a docs scoped change, I didn't want to touch more than was necessary.

Reply:
I'd agree with scoped changes 🙂

Reply:
I'd agree with scoped changes 🙂